include/baresip.h

```diff
@@ -866,10 +866,25 @@ enum ua_event {
 	UA_EVENT_MODULE,
 	UA_EVENT_END_OF_FILE,
 	UA_EVENT_CUSTOM,
+	UA_EVENT_SIPSESS_CONN,
 
 	UA_EVENT_MAX,
 };
 
+
+struct bevent {
```

Comment:
you should also consider to make this struct opaque, i.e. defined in core.h or a .c file
it will make the api more robust and future proof.
and it will be more difficult for API users to make accidental mistakes